management/form.go

```diff
@@ -0,0 +1,132 @@
+package management
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+)
+
+// Form represents an Auth0 form resource.
+//
+// See: https://auth0.com/docs/customize/forms
+type Form struct {
+	// ID is the unique identifier for the form.
+	ID *string `json:"id,omitempty"`
+
+	// Name is the name of the form.
+	Name *string `json:"name,omitempty"`
+	// Description provides details about the form.
+	Description *string `json:"description,omitempty"`
+	// Languages contains the languages of the form.
+	Languages *FormLanguages `json:"languages,omitempty"`
+	// Style contains the style of the form.
+	Style *FormStyle `json:"style,omitempty"`
+
+	// FlowCount is the count of flows in the form.
+	FlowCount *int `json:"flow_count,omitempty"`
+	// Messages contains custom and error messages for the form.
+	Messages *FormMessages `json:"messages,omitempty"`
```

Comment:
I see we have the structs defined for these.
FormStyle, FormMessages, FormLanguages in form.go

Reply:
I am saying I have seen references in docs where many places it's just {} but in some places it's having those attributes so are we sure its just going to be these attributes in them or it can be more.